lib/RestructureCFG/SelectScopePass.cpp

```diff
@@ -77,12 +77,13 @@ static size_t electMaxScopeID(SmallSetVector<BasicBlock *, 2> &Predecessors,
 class SelectScopePassImpl {
   Function &F;
   PostDomTreeOnView<BasicBlock, Scope> PDT;
+  ScopeGraphBuilder SGBuilder;
```

Comment:
This can be `const` too.

Reply:
I made this member `const` here https://github.com/revng/revng/blob/3a9379ef740d1d2b9f7bcad02cc962719251e02a/lib/RestructureCFG/SelectScopePass.cpp#L80